src/cmd/encoder/eventhandler/consumer.go

```diff
@@ -28,15 +27,15 @@ func ConsumeEvents(amqpClientVideoUpload clients.IAmqpClient, s3Client clients.I
 	// and never go back inside.
 	for {
```

Comment:
Not your fault but this function doesn't feel right, I suggest with work later on a refactor together

Reply:
Yep, and same thing for same function on api side